terraform/modules/fake_server_resources/fake_server_resources.tf

```diff
@@ -224,6 +277,133 @@ resource "kubernetes_deployment" "integration-tester" {
   }
 }
 
+# Dead letter topic to which undeliverable intake batch messages are sent
```

Comment:
I'm guessing that you didn't re-use the `pubsub/pubsub.tf` module because in this case, you don't want to create a topic since you're adding an extra subscription to the existing aggregate topic? And I suppose that refactoring the `pubsub` module to only conditionally create a topic would cause a lot of existing `google_pubsub_topic*` resources to get renamed, which invalidates existing Terraform state?

I have to admit that I can't think of a way out of this that doesn't incur the pain of moving TF resources or avoids duplicating this handful of resources *somewhere*. It almost makes me wish Terraform had a simple preprocessor with an `#include` directive so we could at least delegate the copy-pasting to a program.

Reply:
Let's stick with this. All the "better" options would incur some operational pain.